apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -14,16 +14,25 @@ <h1>View Users</h1>
   <hr />
   <p style="margin-bottom: 30px">View submitted users.</p>
   <hr />
-  <div class="filter-container">
+  <div class="filter-container" style="display: flex; gap: 10px;">
     <div class="filter-content">
       <span><b>View Users by Organization: </b></span>
       <select id="organizationFilter" class="organization-filter" onchange="filterTableByOrganization()">
         {% for option in filter_options %}
-          <option class="dropdown-text" {% if option == selected_filter %}selected{% endif %}>{{ option }}</option>
+          <option class="dropdown-text" {% if option == selected_org %}selected{% endif %}>{{ option }}</option>
         {% endfor %}
       </select>
     </div>
+    <select id="statusFilter" onchange="filterTableByStatus()">
```

Comment:
This drop down will need a label, like the organization filter above.

Reply:
Cool. Thanks for the feedback.